drivers/SmartThings/zigbee-smoke-detector/profiles/smoke-battery-aqara.yml

```diff
@@ -0,0 +1,18 @@
+name: smoke-battery-aqara.yml
+components:
+- id: main
+  capabilities:
+  - id: smokeDetector
+    version: 1
+  - id: audioMute
+    version: 1
+  - id: stse.selfCheck
+    version: 1
+  - id: battery
+    version: 1
+  - id: firmwareUpdate
+    version: 1    
+  - id: refresh
+    version: 1    
+  categories:
+  - name: SmokeDetector
```

Comment:
Hi, @Hongming6 

Please add below codes to use specific device presentation.

Details plesae refer to [visibleCondition](https://community.smartthings.com/t/st-edge-vid-defined-in-profile-doesnt-work/277777)
```
  categories:
  - name: SmokeDetector
metadata:
  mnmn: SmartThingsCommunity
  vid: b61c7b6b-10ab-34d7-9bd5-3c0912ce129f
```

Reply:
Thanks,Modified.